### DISCUSSION TITLE
volume appli osmand

### DISCUSSION THREAD

**Discussion by lajemi** [osmandapp/OsmAnd#15596] [Maps Related]
Bonjour, j'ai testé l'appli avec mon camping-car aujourd' hui. j' en suis satisfait, seul problème, le volume. j' ai mis en Bluetooth avec de la musique, lorsque le GPS me donne une indication, le volume de la musique baisse mais le volume du GPS est si bas qu' il est inaudible. Donc je dois choisir, soit je mets le volume à fond pour entendre les indications du GPS, soit je mets la musique au volume normal sans pouvoir entendre les indications... Je n' ai rien trouvé dans les réglages pour solutionner le problème.
Merci de votre retour!
Jean-Michel

**Comment by arncor** [comment]
Salut, t'es allé voir dans le réglage de ton profil / réglages de navigation / voice prompt ?
(j'utilise l'app en anglais donc les noms des menus sont approximatifs).
